modules/t/Utils/FlatFile.t

```diff
@@ -24,15 +24,77 @@ use Test::Exception;
 use Bio::EnsEMBL::Compara::Utils::FlatFile;
 
 subtest 'map_row_to_header' => sub {
```

Comment:
It makes sense to expand the set of tests here. That said, I'm not sure if all of these additional tests are needed; I've commented on the ones I'm not sure about.